etcdmain/config.go

```diff
@@ -177,6 +178,7 @@ func NewConfig() *config {
 		// Should never happen.
 		plog.Panicf("unexpected error setting up clusterStateFlag: %v", err)
 	}
+	fs.BoolVar(&cfg.strictReconfigCheck, "strict-reconfig-check", false, "Reject member change if a number of running node is less than quorum for avoid deadlock in a case of wrong peer URLs")
```

Comment:
Reject to adding member if the number of started nodes is less than quorum.

I would not talk about specific use case of this check here. As long as our users could know what check does, they could understand what it is for. And the check could reject other unreasonable use cases.